src/main/scala/strawman/collection/immutable/Seq.scala

```diff
@@ -93,6 +93,18 @@ trait SeqOps[+A, +CC[_], +C] extends collection.SeqOps[A, CC, C] {
     */
   def updated[B >: A](index: Int, elem: B): CC[B] = fromIterable(View.Updated(coll, index, elem))
 
+  /** Produces a new $coll where a slice of elements in this $coll is replaced by another sequence.
+    *
+    *  @param  from     the index of the first replaced element
+    *  @param  other    the replacement sequence
+    *  @param  replaced the number of elements to drop in the original $coll
+    *  @tparam B        the element type of the returned $coll.
+    *  @return          a new $coll consisting of all elements of this $coll
+    *                   except that `replaced` elements starting from `from` are replaced
+    *                   by `other`.
+    */
+  def patch[B >: A](from: Int, other: IterableOnce[B], replaced: Int): CC[B] = fromIterable(new View.Patched(coll, from, other, replaced))
```

Comment:
No, as I understand it `replaced = 0` means that the other collection is injected into the original collection at index `from` (but it does not replace any elements from the rest of the original collection, whereas `replaced = 3` would mean that the other collection is injected into the original collection at index `from` and the first `replaced` elements from the original are replaced by the other collection (which is always injected in its entirety regardless of `replaced`).

This is also how it works in Scala 2.12.3:
```
scala> List(1, 2, 3).patch(0, List(7, 8, 9), 0)
res0: List[Int] = List(7, 8, 9, 1, 2, 3)

scala> List(1, 2, 3).patch(0, List(7, 8, 9), 2)
res1: List[Int] = List(7, 8, 9, 3)

scala> List(1, 2, 3).patch(2, List(7, 8, 9), 0)
res2: List[Int] = List(1, 2, 7, 8, 9, 3)

scala> List(1, 2, 3).patch(2, List(7, 8, 9), 2)
res3: List[Int] = List(1, 2, 7, 8, 9)
```